enumerable_methods.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+module Enumerable
+  # my_each
+  def my_each
+    return .to_enum unless block_given?
```

Comment:
**[REQUIRED]**
Let's change `return .to_enum unless block_given?` to `return self.to_enum unless block_given?` to solve this error 
![rev6](https://user-images.githubusercontent.com/44072711/69449921-f7c62680-0d29-11ea-83ae-43ffa3b6e706.png)